data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_any_basics.py

```diff
@@ -54,26 +57,23 @@
     ON (concept_id = ancestor_concept_id)
   JOIN `{{project}}.{{dataset}}.observation`
     ON (descendant_concept_id = observation_concept_id)
-  JOIN `{{project}}.{{dataset}}._mapping_observation`
-    USING (observation_id)
   WHERE concept_class_id = 'Module'
     AND concept_name IN ('The Basics')
-    AND src_hpo_id = 'rdr'
     AND questionnaire_response_id IS NOT NULL)
 """)
 
 
-class DropParticipantsWithoutPPI(DropMissingParticipants):
+class DropParticipantsWithoutAnyBasics(DropMissingParticipants):
     """
-    Drops participants without basics survey or EHR data
+    Drops participants who have not completed any of the "The Basics" survey
     """
 
     def __init__(self,
                  project_id,
                  dataset_id,
                  sandbox_dataset_id,
                  table_namer='stage_less'):
-        desc = (f'Sandbox and remove PIDs with no PPI basics or EHR data.'
+        desc = (f'Sandbox and remove PIDs with no PPI basics.'
                 f'Use drop missing participants CR to remove their records.')
 
         super().__init__(issue_numbers=ISSUE_NUMBERS,
```

Comment:
The class initializer needs to be refined.  Line 81 should be updated to 
```python
                         affected_datasets=[cdr_consts.RDR],
```

Lines 86-87 should become
```python
                         table_namer=table_namer)
```

This also means you can drop the import for `RemoveParticipantDataPastDeactivationDate`.